pages/design-tokens/typesetting/font-family.md

```diff
@@ -156,10 +156,10 @@ Role-based tokens set the font family value based on the _role_ the face plays i
 </div>
 
 {:.bg-gold-20v.padding-2.radius-md}
-**Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. This documentation is coming soon. See the inline documentation in `_uswds-theme-typography` for more details.
+**Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. This documentation is coming soon. Create a new project-specific theme settings file. If your project already has a single project-specific theme settings file, you’re all set. If not, create a new file in your project theme file called `_uswds-theme.scss`.
```

Comment:
Omit: "this documentation is coming soon," since we can't promise when "soon" is, and when it does come, we'll surely forget about this reference :'D

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/1866/commits/9eb53229860b3c410608f7006660392c04c34a80